packages/charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -299,27 +298,26 @@ export class VerticalStackedBarChartBase extends React.Component<
         dots.push(
           <circle
             key={`${index}-${subIndex}-dot`}
-            cx={isNumeric ? xScale(circlePoint.x) : xBarScale(circlePoint.index) + this._additionalSpace}
+            cx={
+              isNumeric
+                ? xScale(circlePoint.x.xAxisPoint as number)
+                : xBarScale(circlePoint.index) + this._additionalSpace
+            }
             cy={yScale(circlePoint.y)}
             onMouseOver={
               isLegendSelected && selectedLegendTitle === item
-                ? this._linehover.bind(this, circlePoint)
-                : this._onStackHover.bind(
-                    this,
-                    circlePoint.x,
-                    find(this.props.data, (point: IVerticalStackedChartProps) => point.xAxisPoint === circlePoint.x)
-                      ?.lineData,
-                  )
+                ? this._lineHover.bind(this, circlePoint)
+                : this._onStackHover.bind(this, circlePoint.x)
```

Comment:
@mbest 

it's not working fine
![image](https://user-images.githubusercontent.com/33802398/98312724-539bc500-1ff8-11eb-8a7e-c3c83fda2277.png)

Reply:
Fixed it: 
![image](https://user-images.githubusercontent.com/1230364/98315229-9d4ec600-1f8c-11eb-865d-d90fa37d5e13.png)